syntax = "proto3";

package shell;

message ShellInfo {
    string working_directory = 1;
}